README.md

```diff
@@ -34,34 +32,105 @@
 </p>
 
 <p align="center">
-    <a href="#environment-and-tools">Environment and Tools</a> •
-    <a href="#steps-to-run-and-debug">Steps to run and debug</a> •
+    <a href="#how-to-run">How to run?</a> •
+    <a href="#dependencies">Dependencies?</a> •
+    <a href="#how-to-build">How to build?</a> •
     <a href="#how-to-contribute">How to contribute?</a> •
 </p>
 
-## Environment and tools
+[Versão em Portguês](README.pt.md)
+
+## How to run
+
+You can either download the binary on the [Releases section](https://github.com/zenitheesc/serverd/releases), or you can build the project following the steps bellow
+
+## Dependencies
+
+- [cmake](https://cmake.org/)
+- [nlohmann's json library](https://github.com/nlohmann/json)
+- [cpp-httplib](https://github.com/yhirose/cpp-httplib)
+
+## How to build
+
+First you need to install the necessary dependencies, on ubuntu you can install with this command
```

Comment:
First, you need to install the necessary dependencies. On Debian based systems, you can install them with the following command: